zuul-sample/src/main/resources/log4j2.properties

```diff
@@ -0,0 +1,23 @@
+status = error
+name = Log4j2PropertiesConfig
+
+appenders = console
+
+appender.console.type = Console
+appender.console.name = LogToConsole
```

Comment:
Typically, this naming should be the stream being logged, ex: `stderr`.

Reply:
Updated appender name